src/PrizeVault.sol

```diff
@@ -781,18 +789,36 @@ contract PrizeVault is TwabERC20, Claimable, IERC4626, ILiquidationSource, Ownab
     /// @dev The yield fee balance is included since it's cheaper to keep track of those shares
     ///      internally instead of doing an additional TWAB mint on every liquidation.
     /// @param _totalSupply The total share supply of the vault
+    /// @param _yieldFeeBalance The unrealized yield fee balance
     /// @return The total asset debt of the vault
-    function _totalDebt(uint256 _totalSupply) internal view returns (uint256) {
-        return _totalSupply + yieldFeeBalance;
+    function _totalDebt(uint256 _totalSupply, uint256 _yieldFeeBalance) internal pure returns (uint256) {
+        return _totalSupply + _yieldFeeBalance;
```

Comment:
this function is just adding two parameters!  Is the additional gas cost worth it?  This doesn't seem right to me

Reply:
It's about 40 gas more to call `_totablDebt` compared to adding in-place. The function takes both params in since most functions calling it already have them for other purposes, so that was done to save gas compared to reading `yieldFeeBalance` from storage again.

The function is used 4 times throughout the contract, so it could be managed inline as long as we ensure they all do the same calculation if any further changes are made.